CHANGELOG.md

```diff
@@ -30,6 +30,27 @@ Data.List.Relation.Unary.Unique.Setoid.Properties
 Non-backwards compatible changes
 --------------------------------
 
+
+#### Other
+
+* Renamed a few `-identity` lemmas in `Codata.Stream.Properties` as they were
+  proving two streams bisimilar rather than propositionally equal.
+  ```agda
+  repeat-ap-identity ↦ ap-repeatˡ
+  ap-repeat-identity ↦ ap-repeatʳ
+  ```
+
+* Renamed a few lemmas in `Codata.Stream.Properties` to match the more stdlib
+  conventions:
+  ```agda
+  ap-repeat-commute  ↦ ap-repeat
+  map-repeat-commute ↦ map-repeat
+  ```
```

Comment:
Is it possible to deprecate the old names? We should try to keep our promise of backwards-compatibility as much as possible!